test/provider_os_memory_multiple_numa_nodes.cpp

```diff
@@ -241,7 +256,18 @@ TEST_P(testNumaOnEachNode, checkModeInterleaveSingleNode) {
     EXPECT_NODE_EQ(ptr, numa_node_number);
 }
 
-struct testNumaOnEachCpu : testNuma, testing::WithParamInterface<int> {};
+struct testNumaOnEachCpu : testNuma, testing::WithParamInterface<int> {
+    void SetUp() override {
+        ::testNuma::SetUp();
+
+        int cpuNumber = this->GetParam();
+
+        if (cpuNumber < 0) {
+            GTEST_FAIL() << "get_available_cpus(): sched_getaffinity() return "
```

Comment:
I think this message is too precise - what if we add another ASSERT within `get_available_cpus`? I'd say that we ca write something super generic here, like `something wrong happened when getting CPU count`, and that's ok, because we can always debug the issue and get to the root cause's line (which, in this case, is `sched_getaffinity`)

Reply:
Changed the message